source/_posts/2021-06-21-job-hunting.markdown

```diff
@@ -0,0 +1,69 @@
+
+### Minimizing work disruptions
+Job hunting while working a job is tricky but absolutely doable. There will be some disruption to your day job, but with careful planning and good bookkeeping, you can minimize both that disruption, and the stress and guilt that sometimes comes with it. (Remember, you don't actually want to leave your current workplace in the lurch!)
+
+Step 1 is to accept that you won't be doing your day job at full capacity while interviewing. That's okay. Set expectations at work: keep people updated on your project timelines as they inevitably fall a little behind, and don't bite off more work than you can chew. Ideally, you can make an excuse of the form "I've got some personal stuff going on right now so I'll be working a little under capacity." That's it, that's all they need to know. Resist the instinct to give more details or make more excuses---everyone has lives.
+
+This may be a no-brainer, but make sure that any interviews etc. are marked as "busy" on your work calendar---you don't want people trying to schedule meetings that you then have to decline. I heartily recommend using [Calendly](https://calendly.com/) for scheduling. Spring for the pro plan for a month or two, and you can send individual meeting-booking links to all of the recruiters etc. you're working with, while being sure that nothing scheduled will interfere with anything on your work calendar.
+
+To minimize disruption to your workday, take calls/interviews at the beginning or end of your workday if you can (or during your lunch hour). That said, respect your body's rhythms. If you're foggy in the mornings or if you start to flag in the afternoons, don't schedule technical interviews then.
+
+### Interview Prep
+
+#### Timeline
+If like me, you're *super intimidated* by the prospect of job hunting, remember that you're allowed to start slow. You don't need to be 100% sure you want to leave or completely ready to schedule your interviews to start doing practice problems or to update your resume.
+
+Something that worked well for me: as soon as I realized I would be looking for a new job, I put a recurring weekly 1 hour block of "interview prep" time on my calendar. It helped me make a little progress each week, without freaking out at all times about whether I "should" be prepping.
+
+#### What to Prep
+* **Resume** --- update your resume with all the cool stuff you've been doing lately. Write down all of the interesting projects you've done at this job; you'll only end up putting a few on your resume, but having a list of things you've worked on lately will be helpful for preparing answers to behavioral questions ("tell me about a time a project ran long" etc.).
```

Comment:
The thing I do is write all the things down, then pick and choose based in the company I am applying to.